lang/python/python-mako/monitoring.yml

```diff
@@ -0,0 +1,3 @@
+releases:
+    id: 97183
```

Comment:
I think the actual ID is `3915`. ([ref](https://release-monitoring.org/project/3915/))

97183 seems to be [this](https://release-monitoring.org/project/97183/)